Client/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift

```diff
@@ -603,7 +603,6 @@ extension TopToolbarView: TabLocationViewDelegate {
     
     func tabLocationViewDidLongPressReload(_ tabLocationView: TabLocationView, from button: UIButton) {
         delegate?.topToolbarDidLongPressReloadButton(self, from: button)
-        delegate?.topToolbarDidLongPressLocation(self)
```

Comment:
why this got removed?

Reply:
oh yes this was mistake to call this delegate here, reload button already has its delegate methods linked to it and adding this long press delegate that should be working in address bar creating confusion and wrong behaviour here.